vm/systemSmartContracts/auction.go

```diff
@@ -800,7 +797,7 @@ func (s *stakingAuctionSC) unStake(args *vmcommon.ContractCallInput) vmcommon.Re
 
 	blsKeys, err := getBLSPublicKeys(registrationData, args)
 	if err != nil {
-		s.eei.AddReturnMessage("bls key problem : error " + err.Error())
+		s.eei.AddReturnMessage("bls key problem: error " + err.Error())
```

Comment:
"bls key problem: " + err.Error() is enough

Reply:
done.